core/storage/classifier/gae_models_test.py

```diff
@@ -18,7 +18,7 @@
 
 import datetime
 
-from core.domain import classifier_domain
+from core.domain import classifier_domain  # pylint: disable=invalid-import
```

Comment:
There is no point having a general pragma if we're going to disable it line-by-line.

I think it would be better to have a general policy in the linter script that doesn't check this condition for storage test files (files in core/storage that end in _test.py). Would that be possible? I can see why test files sometimes need the surrounding context that requires reaching into the domain layer.

Reply:
@seanlip Yes that's what I was [asking](https://github.com/oppia/oppia/pull/6180#issuecomment-459828650). 
Now that you've pointed out the need to import the layer in the storage test files, I too agree that disabling the check for such files would be the way.
I'll try to add that check and update you soon accordingly.
Thanks!